apps/api/src/server/app/inspector/inspector.controller.js

```diff
@@ -106,7 +106,7 @@ const validateAppealIdsPresent = function(body) {
 };
 
 const assignAppeals = async function(request, response) {
-	const userId = Number.parseInt(request.headers.userid, 10)
+	const userId = Number.parseInt(request.headers.userid, 10);
```

Comment:
1. not necessary as `validateByUserId` now casts this to int
2. better to use request.get('userId') to read headers, else you had to use the lowercased version of the header name because of how express stores them

Reply:
aha, sneaky